topics/admin/slides/introduction.html

```diff
@@ -155,46 +155,43 @@
 
 Connecting Galaxy to clusters/HPC is covered in the advanced section.
 
-
 ---
 
-# How is Galaxy deployed?
-
-Current:
-
-- Ansible / `git clone https://github.com/galaxyproject/galaxy.git`
-- [Docker](https://github.com/bgruening/docker-galaxy-stable/)
-- *Framework dependencies* provided as Python "wheels", fetched at first startup with `pip`
-- *Tool dependencies* provided as Conda packages or legacy Tool Shed packages
-
-Future:
-
-- Galaxy wheel in PyPI
-
----
-# Making Plans
+# Making plans
 
 **Before** deploying your first Galaxy server:
 
-- Get PostgreSQL (you do not want to switch later)
 - Figure out where Galaxy will be stored
   - Make sure it will be accessible to any eventual compute
 - Figure out where data will be stored
   - Make sure it will be accessible to any eventual compute
 
 ---
+
+# Galaxy deployment options
+
+- `git clone https://github.com/galaxyproject/galaxy.git`
+- [Docker](https://github.com/bgruening/docker-galaxy-stable/)
+- Configuration management (e.g. Ansible)
```

Comment:
So this has a very different tact. The previous list explained more "where do all the parts" come from which I find valuable? I think both of these slides can co-exist. Any reason for moving ansible to the bottom? I think there are other routes to encourage "get started easily" and here maybe we can advocate for potential best long term solution?

Reply:
It was PR https://github.com/galaxyproject/training-material/pull/1996 that led me to change this, Docker added more confusion to a slide that was already mixing apples and oranges. When I went through all the slides, the framework and tool dependencies (and the missing client dependencies) didn't really fit with the "story", which I'd summarise as:

1) Where can Galaxy run?
2) OK you want to install your own Galaxy
3) Software and hardware requirements for the server and the compute
4) How to deploy:
   - Dev -> git
   - Distributable -> Docker
   - Production -> Configuration management
5) Best practices

Which I think also explains why I put Ansible last, to connect to the following slides about Best practices.

Reply:
Ok, sounds good @ removing the "how is it built" just for the "how to deploy" story.

Maybe could you make the dev/distributable/prod distinction a bit clearer in that slide? It read to me like a "best to worst" list at the start.

Reply:
Sure, I was thinking to do that in fact! Would you prefer in the slide or as speaker notes?

Reply:
Oh, great :) I think I lean to preferring it directly in the slides?